docs/application/dotnet/get-started/iot/first-app.md

```diff
@@ -0,0 +1,221 @@
+
+When your application code is ready, build the application. The building process performs a validation check and compiles your files. You must sign the application package with an author certificate when building the application. If you have not yet registered a Tizen certificate in Visual Studio, see [Certificate Manager](../../../vstools/tools/certificate-manager.md).
+
+There are two different ways to build the application:
+
+-   In the Visual Studio menu, select **Build \> Build Solution**.
+-   In the **Solution Explorer** view, right-click the solution name and select **Build**.
+
+Tizen .NET applications are always deployed as installable packages. The package files have the `.tpk` file extension, and the process of generating a package is controlled by the [manifest file](../../../vstools/tools/manifest-editor.md). The Visual Studio template generates the manifest file (`tizen-manifest.xml`) to the top level of the **\<projectname\>.Tizen project**.
+
+For this example application, the default manifest is sufficient. If you want to make any change in the application, such as changing the application icon or installing resources that are used by the application at runtime, see [Package Your Application](#package-your-application).
+
+After you build the application, deploy and run it.
+
+## Deploy and Run Your Application
+
+To run the UI application, you must first deploy it to the reference board with the headed image installed. Deploying means transferring the package file (`.tpk`) to the target and invoking the Tizen Package Manager to install it.
+
+To deploy and run the application on the reference board:
+
+1. Connect the reference board to your PC.
+
+2. Once you connect the reference board, you can deploy the application by clicking the reference board instance in the Visual Studio toolbar. Make the Tizen project as a start-up project with **Set as StartUp Project**.
+
+    ![Deploy your package](media/vs_device_launch.png)
+
+    In the Visual Studio toolbar, you can select the reference board from the drop-down list to change the deployment target.
+
+
+3. If the deployment is successful, the application icon is visible on the device screen. Click the icon to launch the application.
+
+    The following figure shows the launched application on the reference board:
+
+    ![Application running on the reference board](media/cs_first_building_device.png)
+
+Visual Studio uses the Smart Development Bridge (SDB) to communicate with the reference board. If you encounter a problem with detecting the device in Visual Studio, you can check the SDB manually:
+
+1.  In the Visual Studio menu, select **Tools \> Tizen \> Tizen Sdb Command Prompt**.
+2.  In the command prompt, enter `sdb devices`.
+
+    ![Device detection](media/cs_first_building_sdb_prompt.png)
```

Comment:
We should use a real target for IoT. That's because there is no emulator for IoT.
So IP should be displayed. But this IP is not an external real IP. It is an internal IP.
So I think it's no problem to use it.

You can find a internal IP in other documents.
ex) https://developer.tizen.org/development/training/.net-application/test-tizen-.net-application-on-gear-device